app/controllers/people/index.js

```diff
@@ -1,28 +1,3 @@
 import Controller from '@ember/controller';
-import { tracked } from '@glimmer/tracking';
 
-export default class PeopleIndexController extends Controller {
-  @tracked shortedModel;
-
-  constructor() {
-    super(...arguments);
-    this.getAdminUnitInfo()
-  }
-
-  async getAdminUnitInfo() {
-    //let shortedModel = [];
-
-    const mandatories = await this.model.mandatories;
```

Comment:
Resolving relationships like this shouldn't be needed, and if you want to do it, I would do it in the model hook of the route instead.